notebooks/openfiles.sh

```diff
@@ -4,7 +4,7 @@
 group_data_folder="../../group-data"
 
 # Find all the basic_info.json files
```

Comment:
why not do this in python?

Reply:
I was learning bash script for workflow so I used it. Will do in python if needed